pkgs/development/compilers/llvm/17/mlir/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib, stdenv, llvm_meta
+, buildLlvmTools
+, monorepoSrc, runCommand
+, cmake
+, ninja
+, libxml2
+, libllvm
+, version
+, doCheck ? !stdenv.hostPlatform.isMusl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mlir";
+  inherit version doCheck;
+
+  # Blank llvm dir just so relative path works
+  src = runCommand "${pname}-src-${version}" {} ''
```

Comment:
```suggestion
  src = runCommand "mlir-src-${version}" {} ''
```

Reply:
I followed the same pattern from other packages.